src/app/modules/project/embed-button/embed-button.component.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Digital Excellence Copyright (C) 2020
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Lesser General Public License as published
+ *   by the Free Software Foundation version 3 of the License.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty
+ *   of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *   See the GNU Lesser General Public License for more details.
+ *
+ *   You can find a copy of the GNU Lesser General Public License
+ *   along with this program, in the LICENSE.md file in the root project directory.
+ *   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+ */
+
+import { EmbedService } from 'src/app/services/embed.service';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { HttpErrorResponse } from '@angular/common/http';
+import { EmbeddedProject } from 'src/app/models/domain/embedded-project';
+import { environment } from 'src/environments/environment';
+import {EmbeddedProjectResource} from "../../../models/resources/embedded-project-resource";
```

Comment:
Non absolute import & also weirdly formatted. I would recommend enabling format on save in your IDE